joomla/joomla-cms
# Joomla 4.0 updated to PHP 7.2.5 as minimum

Pull Request for Issue https://volunteers.joomla.org/departments/production/reports/1128-production-dept-meeting-november-05-2019

### Summary of Changes

As of https://volunteers.joomla.org/departments/production/reports/1128-production-dept-meeting-november-05-2019 the minimum PHP for Joomla 4.0 is 7.2.5

### Testing Instructions

Install on php lower than 7.2.5 -> see the message
install on php above 7.2.5 -> (and meet the other requirements) see no message

### Expected result

php 7.2.5 is minimum

### Actual result

php 7.2.0 is minimum

### Documentation Changes Required

none as this is a 4.0 change to be documented with the 4.0 PR that will follow
